iceoryx_posh/include/iceoryx_posh/runtime/service_discovery.hpp

```diff
@@ -41,24 +44,13 @@ enum class ServiceDiscoveryEvent : popo::EventEnumIdentifier
 class ServiceDiscovery
 {
   public:
-    ServiceDiscovery() noexcept = default;
+    ServiceDiscovery() = default;
```

Comment:
Why did you remove `noexcept`? IMHO we should not do this now but as part of #1032

Reply:
It will not build in the tests for some GTest reason ...